tools/ocean/argo_getdata.xml

```diff
@@ -0,0 +1,349 @@
+                    <option value="DOXY_ADJUSTED_QC">DOXY_ADJUSTED_QC</option>
+                    <option value="DOXY_DATA_MODE">DOXY_DATA_MODE</option>
+                    <option value="DOXY_QC">DOXY_QC</option>
+                    <option value="NITRATE">NITRATE</option>
+                    <option value="NITRATE_ADJUSTED">NITRATE_ADJUSTED</option>
+                    <option value="NITRATE_ADJUSTED_ERROR">NITRATE_ADJUSTED_ERROR</option>
+                    <option value="NITRATE_ADJUSTED_QC">NITRATE_ADJUSTED_QC</option>
+                    <option value="NITRATE_DATA_MODE">NITRATE_DATA_MODE</option>
+                    <option value="NITRATE_QC">NITRATE_QC</option>
+                    <option value="PH_IN_SITU_TOTAL">PH_IN_SITU_TOTAL</option>
+                    <option value="PH_IN_SITU_TOTAL_ADJUSTED">PH_IN_SITU_TOTAL_ADJUSTED</option>
+                    <option value="PH_IN_SITU_TOTAL_ADJUSTED_ERROR">PH_IN_SITU_TOTAL_ADJUSTED_ERROR</option>
+                    <option value="PH_IN_SITU_TOTAL_ADJUSTED_QC">PH_IN_SITU_TOTAL_ADJUSTED_QC</option>
+                    <option value="PH_IN_SITU_TOTAL_DATA_MODE">PH_IN_SITU_TOTAL_DATA_MODE</option>
+                    <option value="PH_IN_SITU_TOTAL_QC">PH_IN_SITU_TOTAL_QC</option>
+                    <option value="PLATFORM_NUMBER">PLATFORM_NUMBER</option>
+                    <option value="POSITION_QC">POSITION_QC</option>
+                    <option value="PRES">PRES</option>
+                    <option value="PRES_ADJUSTED">PRES_ADJUSTED</option>
+                    <option value="PRES_ADJUSTED_ERROR">PRES_ADJUSTED_ERROR</option>
+                    <option value="PRES_ADJUSTED_QC">PRES_ADJUSTED_QC</option>
+                    <option value="PRES_DATA_MODE">PRES_DATA_MODE</option>
+                    <option value="PRES_QC">PRES_QC</option>
+                    <option value="PSAL">PSAL</option>
+                    <option value="PSAL_ADJUSTED">PSAL_ADJUSTED</option>
+                    <option value="PSAL_ADJUSTED_ERROR">PSAL_ADJUSTED_ERROR</option>
+                    <option value="PSAL_ADJUSTED_QC">PSAL_ADJUSTED_QC</option>
+                    <option value="PSAL_DATA_MODE">PSAL_DATA_MODE</option>
+                    <option value="PSAL_QC">PSAL_QC</option>
+                    <option value="TEMP">TEMP</option>
+                    <option value="TEMP_ADJUSTED">TEMP_ADJUSTED</option>
+                    <option value="TEMP_ADJUSTED_ERROR">TEMP_ADJUSTED_ERROR</option>
+                    <option value="TEMP_ADJUSTED_QC">TEMP_ADJUSTED_QC</option>
+                    <option value="TEMP_QC">TEMP_QC</option>
+                    <option value="TIME_QC">TIME_QC</option>
+                </param>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="output_argo" from_work_dir="argo_data.nc" format="netcdf" label="Argo data"/>
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1">
```

Comment:
more tests would be great, especially to utilize the different modes

Reply:
will do !